index.bs

```diff
@@ -28,6 +28,72 @@ Blah blah blah.
 
 This specification depends on the Infra standard. [[!INFRA]]
 
+<h2 id="deployed-mitigations">Deployed Mitigations</h2>
+
+Some browsers have deployed and announced protections against
+navigational-tracking. This section is a work in progress to detail what
+protections have been shipped and / or are planned. This section should not
+be understood to be comprehensive.
+
+<h3>Safari</h3>
+Safari uses an algorithmic approach to combat navigational-tracking. Safari
+labels a site as having navigational-based cross-site tracking capability if
+the following criteria are met:
+
+<ul>
+  <li>The site has not served content as a third-party.</li>
+  <li>The site automatically redirects the user to another site, immediately
+      or after a short delay.</li>
+  <li>The site has not received a user activation.</li>
+</ul>
+
+Sites can be "tainted" as having cross-site tracking capabilities if they
+redirect to sites already classified as having cross-site tracking capabilities.
+For example, consider the case of a user clicking on a link on
+<code>start.example</code>, which redirects to <code>second.example</code>,
+which redirects to <code>third.example</code>, which redirects to
+<code>end.example</code>. If Safari has classified <code>third.example</code> as
+having tracking capabilities, the above behavior may result in Safari classifying
+<code>second.example</code> as having cross-site tracking capabilities.
+
+Safari deploys defenses against navigational-tracking if a navigation matches
+all of the following criteria:
+
+<ul>
+  <li>The site (determined by eTLD+1) of the user is navigating <em>from</em>
+      has been classified as having cross-site tracking capabilities</li>
+  <li>The URL the user is navigating <em>to</em> has either query parameters
+      or a URL fragment</li>
```

Comment:
@johnwilander I vaguely remember you mentioning that the target site could prevent its cookies from having their lifetime capped by doing a server-side redirect that removes the query parameters. Am I remembering correctly?